aspnetcore/mvc/razor-pages/index.md

```diff
@@ -58,26 +58,26 @@ Run `dotnet new razor` from the command line.
 
 Razor Pages is enabled in *Startup.cs*:
 
-[!code-cs[main](index/sample/RazorPagesIntro/Startup.cs?name=Startup)]
+[!code-cs[main](index/sample/RazorPagesIntro/Startup.cs?name=snippet_Startup)]
 
 Consider a basic page:
 <a name="OnGet"></a>
 
 [!code-cshtml[main](index/sample/RazorPagesIntro/Pages/Index.cshtml)]
 
-The preceding code looks a lot like a Razor view file. What makes it different is the `@page` directive. `@page` makes the file into an MVC action - which means that it handles requests directly, without going through a controller. `@page` must be the first Razor directive on a page. `@page` affects the behavior of other Razor constructs. The [@functions](xref:mvc/views/razor#functions) directive enables function-level content.
+The preceding code looks a lot like a Razor view file. What makes it different is the `@page` directive. `@page` makes the file into an MVC action - which means that it handles requests directly, without going through a controller. `@page` must be the first Razor directive on a page. `@page` affects the behavior of other Razor constructs.
 
-A similar page, with the `PageModel` in a separate file, is shown in the following two files. The *Pages/Index2.cshtml* file:
+A similar page, using a `PageModel` class, is shown in the following two files. The *Pages/Index2.cshtml* file:
 
 [!code-cshtml[main](index/sample/RazorPagesIntro/Pages/Index2.cshtml)]
 
-The *Pages/Index2.cshtml.cs* 'code-behind' file:
+The *Pages/Index2.cshtml.cs* "code-behind" file:
 
 [!code-cs[main](index/sample/RazorPagesIntro/Pages/Index2.cshtml.cs)]
 
 By convention, the `PageModel` class file has the same name as the Razor Page file with *.cs* appended. For example, the previous Razor Page is *Pages/Index2.cshtml*. The file containing the `PageModel` class is named *Pages/Index2.cshtml.cs*.
 
-For simple pages, mixing the `PageModel` class with the Razor markup is fine. For more complex code, it's a best practice to keep the page model code separate.
+For simple pages, mixing the `PageModel` class with the Razor markup is fine. This is accomplished by defining the page model and its members inside a [@functions](xref:mvc/views/razor#functions) block. For more complex code, it's a best practice to keep the page model code separate.
```

Comment:
Let's remove this. The feedback has been pretty strong that we should only show best practices. There should be no examples of using `@functions` for the page model.